### PR TITLE
Clear Cart fixed

### PR DESCRIPTION
**Pull request by gaurab-khanal**
Issue: Weird Size of Call to Action Buttons on Smaller Devices #14

### CODE REVIEW

File: client/src/Components/Cart.jsx
Comment (suryanshsingh2001) [inline client/src/Components/Cart.jsx:1]: Please change `class` to `className`.

**Review by suryanshsingh2001 — CHANGES_REQUESTED** [review]
Please change `class` to `className`

**Review by suryanshsingh2001 — APPROVED** [review]

**Comment by suryanshsingh2001** [comment]
You can resolve the issues solve here #18.

**Comment by gaurab-khanal** [comment]
> You can resolve the issues solve here #18.

Should I need to make new PR to fix #18 issue?

**Pull request merged**